File: read_file.js
const fs = require('fs');
const content = fs.readFileSync('e:\\Anirban Full backend\\final\\src\\pages\\SetQuestions.jsx', 'utf-8');
const lines = content.split('\n');

// Print lines 880-920
console.log("Context around line 887:\n");
for (let i = 879; i < Math.min(920, lines.length); i++) {
  console.log(`${i+1}: ${lines[i]}`);
}

File: read_file.py
import os
import sys

filepath = r"e:\Anirban Full backend\final\src\pages\SetQuestions.jsx"
if not os.path.exists(filepath):
    print(f"File not found: {filepath}")
    sys.exit(1)

with open(filepath, 'r', encoding='utf-8') as f:
    lines = f.readlines()

# Print lines 880-920
print("Context around line 887:\n")
for i in range(879, min(920, len(lines))):
    print(f"{i+1}: {lines[i]}", end='')
